viam-cartographer/src/slam_service/slam_service.cc

```diff
@@ -579,77 +579,58 @@ void SLAMServiceImpl::GetLatestSampledPointCloudMapString(
         }
     }
 
+    // Get data from painted surface in RGBA format
     auto painted_surface = painted_slices->surface.get();
     int width = cairo_image_surface_get_width(painted_surface);
     int height = cairo_image_surface_get_height(painted_surface);
-    // Get all pixels from the painted surface in RGBA format
-    auto data = cairo_image_surface_get_data(painted_surface);
-
-    // Total number of bytes in image (4 bytes per pixel)
-    int size_data = width * height * 4;
-
-    // Each pixel contains 4 bytes of information in RGBA format
-    // data_vect[i + 0] is the R channel
-    // data_vect[i + 1] is the B channel
-    // data_vect[i + 2] is the G channel
-    // data_vect[i + 3] is the A channel
-    std::vector<unsigned char> data_vect(data, data + size_data);
-
+    auto image_data = cairo_image_surface_get_data(painted_surface);
+    // // Each pixel contains 4 bytes of information in RGBA format
+    // // data_vect[i + 0] is the R channel
+    // // data_vect[i + 1] is the B channel
+    // // data_vect[i + 2] is the G channel
+    // // data_vect[i + 3] is the A channel
```

Comment:
Since `data_vect` no longer exist, should this comment move somewhere else?

Also, we can probably remove the double `//`